src/components/Button/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import './button.scss';
+
+export const Button = ({ props, text, styles }) => (
+  <div>
+    <button type="button" onClick={props} className={styles}>
```

Comment:
```suggestion
    <button 
       type="button" 
       onClick={props} 
       className={styles}
     >
```

Більше 2х тегів =>  перенось

Reply:
```suggestion
    <button type="button" onClick={onClick} className={styles}>
```